python/pyspark/sql/functions.py

```diff
@@ -1839,7 +1839,8 @@ def from_json(col, schema, options={}):
     string.
 
     :param col: string column in json format
-    :param schema: a StructType or ArrayType of StructType to use when parsing the json column
+    :param schema: a StructType or ArrayType of StructType to use when parsing the json column.
+                   Since Spark 2.3, the DDL-formatted string is also supported for the schema.
```

Comment:
Up to my knowledge, we don't have a specific rule for describing additional behaviours in Python documentation (at least I see such changes were merged time to time). I tried to make this prettier as below.